docs/install/install-node.rst

```diff
@@ -58,9 +60,17 @@ Installing on CentOS 7
    .. task:: install_flocker centos-7
       :prompt: [root@centos]#
 
-#. **Repeat steps 1 and 2 for all other nodes:**
+#. **Install the** ``clusterhq-flocker-docker-plugin`` **package:**
+
+   Run the following command as root on the target node:
+
+   .. prompt:: bash [root@centos]#
+   
+      yum install -y clusterhq-flocker-docker-plugin
```

Comment:
This step depends on the commands for installing `flocker-node` having been run. Emphasise that the flocker-node steps are required, but the Flocker plugin is an optional additional step.